apps/web/components/v2/apps/CalendarListContainer.tsx

```diff
@@ -166,8 +166,9 @@ function ConnectedCalendarsList(props: Props) {
                       <div className="w-32">
                         <DisconnectIntegration
                           credentialId={item.credentialId}
-                          label={t("remove")}
+                          trashIcon
                           onSuccess={props.onChanged}
+                          buttonProps={{ className: "border border-gray-300" }}
```

Comment:
<img width="529" alt="CleanShot 2022-09-15 at 11 14 27@2x" src="https://user-images.githubusercontent.com/55134778/190378500-a6f13b07-ae45-4ce9-926d-0a904999e82e.png">
Can we have this the same style as the other disconnect button in settings? 

I believe  the one above will not be square

Reply:
@sean-brydon well, actually, it does look correct, I tweaked `DisconnectIntegration` to show as I needed when no label was passed and the trashIcon is present. See: 
![ezgif-2-128e20084a](https://user-images.githubusercontent.com/467258/190380183-4d05e170-9450-4443-ac8b-215105df02dc.gif)

Reply:
Oh that's perfect then :) My apologies - nice job @leog